pylivetrader/backend/alpaca.py

```diff
@@ -263,7 +268,10 @@ def order(self, asset, amount, style):
             zp_order = self._order2zp(order)
             return zp_order
         except APIError as e:
-            log.warning('order is rejected {}'.format(e))
+            log.warning('order for symbol ${} is rejected {}'.format(
```

Comment:
How about adding order id? Also I'm not a big fan of cashtag here. (I do like it personally, it's just not best place here)

Reply:
Oh but if it's rejected, we don't have id..